modules/ROOT/pages/install-port-reqs.adoc

```diff
@@ -58,11 +58,11 @@ The following table lists the ports that must be accessible when installing Anyp
 | 4242 | TCP | Internal | Bandwidth checker
 |===
 
-== Port IPs and Hostnames to Whitelist
+== Port IPs and Hostnames to Allowlist
```

Comment:
I would verify here, but I think the verb form is just "to allow" ....I don't think we're using allowlist as a verb, but I don't actually know 😬